ClosedXML/Excel/Ranges/IXLRanges.cs

```diff
@@ -19,6 +19,12 @@ public interface IXLRanges : IEnumerable<IXLRange>, IDisposable
         /// <param name="range">The range to remove from this group.</param>
         void Remove(IXLRange range);
 
+        /// <summary>
+        /// Removes all ranges from the collection optionally disposing them.
+        /// </summary>
+        /// <param name="dispose"></param>
+        void RemoveAll(bool dispose = true);
```

Comment:
What do you think of this rather:
```c#
void RemoveAll(Predicate<IXLRange> match = null, bool releaseEventHandlers = true);
```

to better match the standard Linq `RemoveAll()` method?

And I think `releaseEventHandlers` is a better description for the user than `dispose`. The user won't know what `dispose` means.

Reply:
Of course, if `match` is `null`, then make `match = _ => true`.

Reply:
Or do you think it would cause users to think they can do something like:

```c#
var ranges = new XLRanges();
ranges.Add(ws.Range("A1:Z10"));
ranges.RemoveAll(r => r.RangeAddress.ToString().Equals("B2:C4"));
```

which is obviously not possible without breaking up the ranges first.

Reply:
> What do you think of this rather: ...

I agree.

> And I think releaseEventHandlers is a better description for the user than dispose.

True. Moreover, while I was working on another PR (#766) I realized that this ```dispose``` argument may become meaningless if we got rid of events subscription.

> Or do you think it would cause users to think they can do something like:

I really doubt someone can decide to do this. We have a collection and a method that accepts a predicate - for me, this would be quite obvious that the predicate will be applied to items in the collection only, not to some derivatives.